engine/src/test/scala/cromwell/CromwellTestkitSpec.scala

```diff
@@ -169,7 +184,11 @@ object CromwellTestkitSpec {
     """.stripMargin)
 
   lazy val DefaultLocalBackendConfigEntry = BackendConfigurationEntry(
-    name = "local", className = "cromwell.engine.backend.local.LocalBackend", DefaultLocalBackendConfig)
+    name = "local",
```

Comment:
oh sorry, I thought you had removed the need for this already.  It seemed the key was redundant with `name` and the src/test/application.conf config above appears to be `name`less.  I guess that was not intentional?